.lycheeignore

```diff
@@ -37,3 +37,77 @@ socket://localhost
 
 # Bug on Lychee (see later)
 https://products.espressif.com/
+
+
+# List of invalid links as of 18.10.2024
+
+# ./content/blog/developing-user-interfaces-for-custom-boards-with-esp-microcontrollers/index.md
+https://github.com/espressif/esp-bsp/blob/master/SquareLine/boards/custom_waveshare_7inch/components/ws_7inch/CMakeLists.txt
```

Comment:
@pedrominatel @georgik 

Can you please suggest a good approach to update or fix the invalid links to numerous Espressif repos?

Should the content in the respective articles be reviewed as well to see if it is still relevant?

Reply:
For the LVGL `SquareLine/boards/custom_waveshare_7inch`, we can change it to `SquareLine/boards/v8/custom_waveshare_7inch`.

We can change our GitHub repo links to the project repo root to find them in the new location. If the project does not exist, point to the Espressif Org on GitHub.

https://sqfmi.com/ works fine.

For third-party ones, please keep it on the ignore list.